apiary.apib

```diff
@@ -719,12 +719,35 @@ API endpoints that Operation Code's Rails backend makes available to its React f
 
 + Response 400 (application/json)
 
+### List All Mentors [GET]
```

Comment:
These `Mentor` endpoints will go under their own h2 heading, and alphabetized accordingly.

So not under `Requests`, but under `Mentors`.